frontend/src/components/graphs/HeatMap.js

```diff
@@ -32,15 +39,18 @@ function HeatMap(props) {
   };
 
   return (
-    <div id="chart" className="mt-3">
-      <ReactApexChart
-        options={options}
-        series={data}
-        type="heatmap"
-        height={400}
-        width="100%"
-      />
-    </div>
+    <>
+      <div id="chart" className="mt-3">
+        <ReactApexChart
+          options={options}
+          series={data}
+          type="heatmap"
+          height={400}
+          width="100%"
+        />
+      </div>
+      <CopyGraphButton key={animationEnded} chartId='heatmap' animationEnded={animationEnded}/>
```

Comment:
De copy to clipboard werkt niet op mobile misschien hier {!isMobile && <>} van maken?